packages/core/src/App/Containers/RealAccountSignup/signup-error-content.jsx

```diff
@@ -75,21 +124,33 @@ const ErrorCTA = ({ code, onConfirm }) => {
     }
 };
 
-const SignupErrorContent = ({ message, code, onConfirm }) => {
+const SignupErrorContent = ({ message, code, onConfirm, className, error_field = {} }) => {
+    const is_invalid_field_error = ['InputValidationFailed', 'PoBoxInAddress', 'InvalidPhone'].includes(code);
+    const getIconSize = () => {
+        if (is_invalid_field_error) return '64';
+        else if (code === 'InvalidAccount') return '96';
+        return '115';
```

Comment:
is the size `115` from the design? It doesn't look like a standard icon size.

Reply:
Currently in prod, it is 115 for icons in this component. For this specific scenarios, i have updated the icon size as per design.